src/components/dropdown/status.tsx

```
@@ -97,21 +102,17 @@ const StatusDrop = ({ status, onChange, type }: StatusDropProps) => {
-            취업
-          </DropdownItem>
-          <DropdownItem onClick={() => handleOptionClick('자퇴')}>
-            자퇴
-          </DropdownItem>
+          {type === 'ATTENDANCE'
+            ? attendanceStatus.map((item) => (
+                <DropdownItem onClick={() => handleOptionClick(item)}>
+                  {item}
+                </DropdownItem>
+              ))
+            : homeroomStatus.map((item) => (
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)